### PR TITLE
Adding First Readme to the project

### PR DESCRIPTION
**Pull request by theburi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,59 @@
 # MongoDB Atlas Operator
 Welcome to MongoDB Atlas Operator - the Operator allowing to manage Atlas Clusters from Kubernetes.
+> Current Status: *pre-alpha*. We are currently working on the initial set of features that will give users the opportunity to provision Atlas projects, clusters and database users using Kubernetes Specifications and bind connection information into the applications deployed to Kubernetes.   
+## Quick Start guide
+### Step 1. Deploy Kubernetes operator using all in on config file
```
Comment (vasilevp) [inline README.md:5]:
```suggestion
### Step 1. Deploy Kubernetes operator using all in one config file
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+```
+kubectl apply -f https://raw.githubusercontent.com/mongodb/mongodb-atlas-kubernetes/deploy/all-in-one.yaml
+```
+## Step 2. Create Atlas Cluster
```
Comment (vasilevp) [inline README.md:9]:
```suggestion
### Step 2. Create Atlas Cluster
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+  name: "Test-cluster"
+  projectRef:
+    name: my-project
+  providerSettings:
+    instanceSizeName: M10
+    providerName: AWS
+    regionName: US_EAST_1
+EOF
+```
+## Step 3. Inspect Atlas Cluster Status
```
Comment (vasilevp) [inline README.md:54]:
```suggestion
### Step 3. Inspect Atlas Cluster Status
```

File: README.md (hunk 1 above)
Comment (theburi) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (theburi) [reply README.md]: resolved

**Review by vasilevp — APPROVED** [review]

**Pull request merged**